Jenkinsfile

```diff
@@ -44,7 +44,7 @@ for(i = 0; i < buildTypes.size(); i++) {
                             // The -Dmaven.repo.local=${pwd()}/.repository means that Maven will create a
                             // .repository directory at the root of the build (which it gets from the
                             // pwd() Workflow call) and use that for the local Maven repository.
-                            def mvnCmd = "mvn -Pdebug -U clean install ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository" 
+                            def mvnCmd = "mvn -Pdebug -U clean install javadoc:javadoc ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository" 
```

Comment:
Theoretically. In practice it is unusual for individually passing changes to merge to a breaking change, and in particular it seems unlikely to cause a Javadoc error, which is mostly sensitive to local code (other than changes to members in `@link`). Anyway even if this happened, we would get a clear failure in the `master` job.